eZ/Bundle/EzPublishCoreBundle/Command/ReindexCommand.php

```diff
@@ -44,14 +58,23 @@ public function initialize(InputInterface $input, OutputInterface $output)
      */
     protected function configure()
     {
+        $cpuCores = $this->getNumberOfCPUCores();
         $this
             ->setName('ezplatform:reindex')
-            ->setDescription('Recreate search engine index')
-            ->addOption('iteration-count', 'c', InputOption::VALUE_OPTIONAL, 'Number of objects to be indexed in a single iteration', 20)
+            ->setDescription('Recreate or Refresh search engine index')
+            ->addOption('iteration-count', 'c', InputOption::VALUE_OPTIONAL, 'Number of objects to be indexed in a single iteration, for avoiding using to much memory', 50)
             ->addOption('no-commit', null, InputOption::VALUE_NONE, 'Do not commit after each iteration')
+            ->addOption('no-purge', null, InputOption::VALUE_NONE, 'Do not purge before indexing. BC NOTE: Should this be default as of 2.0?')
```

Comment:
It's strange to see at the end of description line a question about BC ;)
Side note: If in the future this would be default, then we'd need `purge` option to disable it, right?

Reply:
TODO: Remove this note ;)  _(we can still consider making refresh a default behavior in the future, but for now the logic where no-purge is implicit when specifying subset of index is I think ok for now)_